<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEmployeeInfo</class>
 <widget class="QDialog" name="EditEmployeeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LU ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Approved:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nameComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="luidTextField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleTextField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailTextField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneTextField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="approvedComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
